### DISCUSSION TITLE
[Wiki] Elements out of scope

### DISCUSSION THREAD

**Discussion by cyber5tar86** [langroodi/Adaptive-AUTOSAR#12] [Q&A]
From what I can se from the wiki page many elements are out-of-scope; Can I ask you the motivation beneath?
Thank you!

**Comment by langroodi** [comment]
The motivation of the whole project is educational. So, new people to automotive software engineering (including me) may learn something from it (at least I hope).

The motivation of why many functional clusters are out of scope is only time. Personally, I can spend in average 15 hours a week on this project. That means if I am going to cover the whole ARA, I never can finish it alone. So, I've defined a timeline for 4 years and cover the parts that at least I have some background knowledge about it.
